package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "hackerman-bot",
-  "version": "1.0.0",
-  "description": "Hackerman bot",
-  "main": "index.js",
+  "version": "2.0.0",
+  "description": "udspy's discordbot",
```

Comment:
Please fix this, you spelled "Ubspy" wrong as well

Reply:
lol rip